advanced_functionality/autogluon-multimodal-containers/AutoGluon_Multimodal_SageMaker_Containers.ipynb

```diff
@@ -0,0 +1,595 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "519a2c14-1ecf-48f6-b6af-debb4a599345",
+   "metadata": {},
+   "source": [
+    "# AutoGluon Multimodal with Deep Learning Containers on SageMaker\n",
+    "\n",
+    "[AutoGluon](https://github.com/awslabs/autogluon) automates machine learning tasks enabling you to easily achieve strong predictive performance in your applications. With just a few lines of code, you can train and deploy high-accuracy deep learning models on tabular, image, and text data.\n",
+    "This example shows how to use AutoGluon-Multimodal with Amazon SageMaker by applying [pre-built deep learning containers](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#autogluon-training-containers)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9e373e83-6681-4337-8e4c-69768102ddc0",
+   "metadata": {},
+   "source": [
+    "# Prerequisites"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "62626ac2-e29e-4e1f-9cfd-cd448cda56fb",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!python -m pip install --upgrade pip\n",
+    "!python -m pip install -U 'sagemaker>=2.103.0'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a08649e9-e554-4ffc-8c0c-aac66d6d75b9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import base64\n",
+    "import os\n",
+    "import shutil\n",
+    "\n",
+    "import boto3\n",
+    "import sagemaker\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from sagemaker import utils\n",
+    "from sagemaker.serializers import CSVSerializer\n",
+    "\n",
+    "from autogluon.core.utils.loaders import load_zip\n",
+    "\n",
+    "from ag_model import (\n",
+    "    AutoGluonTraining,\n",
+    "    AutoGluonInferenceModel,\n",
+    "    AutoGluonMultimodalPredictor,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef5ac9cf-8bec-4ea9-af42-4fc15763a393",
+   "metadata": {},
+   "source": [
+    "To access SageMaker to train and deploy models, you need to use a IAM role that has adequate SageMaker perssions. If `role` is set to `sagemaker.get_execution_role()`, than the default role for the account profile needs to have the required SageMaker permissions, which is most easily set up with the `AmazonSageMakerFullAccess` policy. If using a different role to manage SageMaker permissions, the `role` variable can be set to the URI for the IAM role with the required SageMaker policy."
```

Comment:
We probably want to inform them that they should scope down their policy instead of `FullAccess`.
Besides that, I think the user would also need S3 permissions

Reply:
Addressed in next revision